src/main.js

```diff
@@ -1,23 +1,20 @@
-import 'bootstrap/dist/css/bootstrap.css'
-import 'bootstrap-vue/dist/bootstrap-vue.css'
+import './styles/legacy_bootstrap.scss'
+import './styles/index.css'
 
 import Vue from 'vue'
 import Vuex from 'vuex'
 import App from './App.vue'
+import VueRouter from 'vue-router'
```

Comment:
This was being imported in router.js before, but that meant we were mutating the global vue when loading our test. This prevented us from mocking the router.